charts/gateway-helm/templates/certgen-rbac.yaml

```diff
@@ -5,8 +5,15 @@ metadata:
   namespace: '{{ .Release.Namespace }}'
   labels:
   {{- include "eg.labels" . | nindent 4 }}
+  {{- if .Values.certgen.rbac.labels }}
+  {{- toYaml .Values.certgen.rbac.labels | nindent 4 }}
+  {{- end }}
   annotations:
+  {{- if .Values.certgen.rbac.annotations }}
+    {{- toYaml .Values.certgen.rbac.annotations | nindent 4 -}}
+  {{ else }}
```

Comment:
why this there a `else` here, can't we just append ?

Reply:
we can give a try, eventually i will open a new issue :)